mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingaccountsummary/SavingsAccountSummaryFragment.java

```diff
@@ -456,4 +488,150 @@ public void onDestroyView() {
         void doTransaction(SavingsAccountWithAssociations savingsAccountWithAssociations, String
                 transactionType, DepositType accountType);
     }
+
+    // Takes ScreenShot of Relative layout
+    public Bitmap layoutToImage() {
```

Comment:
Hey, @PatelVatsalB21 this is no the correct way to do like you are first taking the screenshot of the images and then converting it to pdf and then deleting the image which is not the correct way to create a transition receipt

Reply:
Please go through the method carefully, it doesn't save image, it passes bitmap.

Reply:
yeah but again it is compressing to JPEG and also why you are not doing something which creates PDF automatically ?

Reply:
Also, the pdf contains all the data?

Reply:
@robustTechie this is a pdf generated [ok_123.pdf](https://github.com/openMF/android-client/files/5690070/ok_123.1.pdf), also attached in pr. I think it contains all information shown in savings account summary fragment.